### PR TITLE
Bring gRPC and Digital Twin apis into runtime

### PR DESCRIPTION
**Pull request by splatch**
I attempted to import horizon feature set and it seems to work. With a bit of copy-paste I was able to cut unclear parts. This is I believe first stage which makes sure that grpc and digital twin deploys clearly.

### CODE REVIEW

**Comment by j-white** [comment]
Looks sensible so far

File: grpc/workflow/model/src/main/java/org/opennms/poc/ignite/grpc/model/WorkflowTwin.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.opennms.poc.ignite.grpc.model;
+
+import java.util.UUID;
+
+public class WorkflowTwin {
```
Comment (mfrazier63) [inline grpc/workflow/model/src/main/java/org/opennms/poc/ignite/grpc/model/WorkflowTwin.java:5]: Could Lombok this.

**Review by mfrazier63 — APPROVED** [review]
This looks awesome because you write code like I do. :)

**Comment by splatch** [comment]
I moved model out to separate module in order to use it as a grpc data structure.

File: grpc/workflow/model/src/main/java/org/opennms/poc/ignite/grpc/model/WorkflowTwin.java (hunk 1 above)
Comment (splatch) [reply grpc/workflow/model/src/main/java/org/opennms/poc/ignite/grpc/model/WorkflowTwin.java]: Changed to Workflow from worker.

**Comment by splatch** [comment]
Follow up changes in #14.

**Pull request merged**